chain/client/tests/challenges.rs

```diff
@@ -535,18 +536,16 @@ fn test_verify_chunk_invalid_state_challenge() {
             challenge_body.partial_state.0,
             vec![
                 vec![
-                    1, 7, 0, 92, 241, 96, 67, 27, 175, 62, 116, 3, 39, 175, 167, 179, 91, 63, 212,
-                    212, 75, 174, 160, 30, 148, 184, 11, 249, 27, 202, 188, 201, 221, 145, 255,
-                    115, 118, 86, 148, 43, 154, 46, 88, 27, 131, 172, 99, 25, 223, 149, 122, 104,
-                    247, 21, 42, 198, 205, 43, 239, 65, 133, 166, 38, 174, 254, 133, 217, 171, 30,
-                    7, 228, 175, 99, 17, 113, 5, 94, 136, 200, 39, 136, 37, 110, 166, 241, 148,
-                    128, 55, 131, 173, 97, 98, 201, 68, 82, 244, 223, 70, 86, 143, 134, 2, 0, 0, 0,
+                    1, 5, 0, 195, 214, 6, 46, 119, 169, 1, 3, 121, 138, 244, 191, 143, 67, 22, 114,
```

Comment:
Do you know why this changed?

Reply:
I had to refactor GenesisConfig initialization and now it does not attach `DEFAULT_TEST_CONTRACT` contract by default.